pkg/retagger/job_single.go

```diff
@@ -0,0 +1,116 @@
+package retagger
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/giantswarm/microerror"
+)
+
+// SingleJob is a concrete job which can be executed.
+type SingleJob struct {
+	Source Source
+
+	Destination Destination
+
```

Comment:
If there's meaning to this whitespace, I think it should have a comment and if not, let's get rid of it.